Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG CUDA_TAG=7.0
 ARG OS_TAG=7
-FROM alicevision:centos${OS_TAG}-cuda${CUDA_TAG}
```

Comment:
It should be: `alicevision/alicevision:centos${OS_TAG}-cuda${CUDA_TAG}`

See https://hub.docker.com/r/alicevision/alicevision